tests/utilities/test_seed.py

```diff
@@ -53,3 +54,24 @@ def test_out_of_bounds_seed(seed):
     with pytest.warns(UserWarning, match="is not in bounds"):
         actual = seed_utils.seed_everything(seed)
     assert actual == 123
+
+
+def test_reset_seed_no_op():
+    """ Test that the reset_seed function is a no-op when seed_everything() was not used. """
+    assert "PL_GLOBAL_SEED" not in os.environ
+    seed_before = torch.initial_seed()
+    seed_utils.reset_seed()
+    assert torch.initial_seed() == seed_before
+    assert "PL_GLOBAL_SEED" not in os.environ
+
+
+def test_reset_seed_everything():
+    """ Test that we can reset the seed to the initial value set by seed_everything() """
+    assert "PL_GLOBAL_SEED" not in os.environ
+    seed_utils.seed_everything(123)
+    assert os.environ["PL_GLOBAL_SEED"] == "123"
+    before = torch.rand(1)
+    _ = torch.rand(1)
```

Comment:
Why is this line required?

Reply:
this line is not required. it was a test for the good reviewers with the eagle eye!